Resources/Server Info/RP_Rules.txt

```diff
@@ -0,0 +1,103 @@
+    - Character names should be something that a member of that species would reasonably have.
+    - No famous real or fictional names of characters.
+
+[color=#a4885c]9.[/color] Knowingly using exploits, external programs, or other unintended game mechanics to gain in-game advantages or to lag/crash the server is [color=#ff0000]not allowed[/color].
+
+[color=#a4885c]10.[/color] Multi-keying/Alts on the same server is [color=#ff0000]not allowed[/color].
+	- Admins may ask players in possession of multiple alt accounts to choose one to play on and ban the other accounts.
+	- If you are caught knowingly using alt accounts you may have your main account banned as well.
+
+[color=#a4885c]11.[/color] Using information gained outside the game to your advantage is [color=#ff0000]not allowed[/color]. IC and OOC are to be kept separate at all times.
+	- Metacommunication, or communicating with other players via out of game methods (such as Discord) to gain an unfair advantage is also strictly [color=#ff0000]prohibited[/color].
+    - This is a roleplaying game, thus IC conflicts must only be resolved with IC (Local) chat. Utilizing voice chat, calling across the room, OOC, or other out-of-game communication methods to coordinate in-game actions is forbidden and will result in a [color=#ff0000]appeal-only permanent ban[/color] in most cases.
+    - Using emotes or LOOC to bypass IC chat restrictions (accents, unconsciousness, death) is not allowed.
+    - Preemptive spacing or destruction of high-risk items also falls under this rule.
+
+Security Rules
+
+[color=#a4885c]12.[/color] Security is expected to use non-lethal measures unless faced with lethal force or a situation where non-lethal measures aren’t possible. Ideally, do not enter combat mode, as security stun batons do not deal damage, but still stun, when outside of combat mode.
+    - If you still manage to kill someone as security, get them cloned immediately, and make sure their stuff is returned. Security should make their best effort to keep criminals alive and in custody.
+[color=#a4885c]13.[/color] Prisoners should not be given sentences longer than 10 minutes. Certain extreme exceptions like bombers and assassins may warrant permabrigging, or in the very worst cases, execution.
+    - Note that employment at a hostile corporation (i.e. traitor status) alone does not qualify someone for perma or execution.
+    - Security are required to follow Space Law. If in doubt, consult Centcom’s law team.
+
+MRP Specific Rules
+
+[color=#a4885c]14.[/color] This server is a Roleplay Expected server. We define this as performing your assigned duties. This is a ‘do your job’ server. This means it is important to do what is expected out of your department and not what would be expected out of other departments.
```

Comment:
```suggestion
[color=#a4885c]14.[/color] This server is a "Roleplay Expected" server. We define this as pretending to actually be your character performing your character's assigned duties. This is a ‘do your job’ server; Nanotrasen would not hire the grossly incompetent/psychopathic/etc, so don't act that way.
```

Reply:
UGOHUGHOGUHOGHUGHUGHOGUHGOHUGOHUGHOGUHOGUHOGHGFUHGOHUGOHUGHOGUHOG

Reply:
im not doing these 'bikeshed the wording of an entire sentence' reviews